vendor.conf

```diff
@@ -26,21 +26,21 @@ github.com/sirupsen/logrus v1.4.1
 github.com/urfave/cli 7bc6a0acffa589f415f88aca16cc1de5ffd66f9c
 golang.org/x/net b3756b4b77d7b13260a0a2ec658753cf48922eac
 google.golang.org/grpc v1.12.0
```

Comment:
can we also update gRPC? (though it might be risky.. I've once tried to update grpc for buildkit and it broke something)

Reply:
Yes, was looking at some other dependencies as well, I can have a look at bringing more up to date

Reply:
I'd like to do that in a separate PR though, so that I can work on it when I have  some time in between things 😂

Reply:
Opened https://github.com/containerd/containerd/pull/3192